src/main/java/com/tutrit/bean/Customer.java

```diff
@@ -0,0 +1,8 @@
+package com.tutrit.bean;
```

Comment:
we don't place `src` at project root, please take another look on example at https://github.com/max-java/quickstart-multimodule-maven

Reply:
I  moved this component to the correct directory